changelogs/head.asciidoc

```diff
@@ -16,4 +16,6 @@ https://github.com/elastic/apm-server/compare/8.15\...main[View commits]
 ==== Intake API Changes
 
 [float]
-==== Added
\ No newline at end of file
+==== Added
+
+- APM Server now automatically retries document-level 429s from Elasticsearch to avoid dropping data. This is controlled by `output.elasticsearch.max_retries`, and defaults to `3`. {pull}13620[13620]
```

Comment:
nit: make it clear that `output.elasticsearch.max_retries` controls both request-level and document-level retries.
```suggestion
- APM Server now automatically retries document-level 429s from Elasticsearch to avoid dropping data. `output.elasticsearch.max_retries` now controls both request-level and document-level retries, and defaults to `3`. {pull}13620[13620]
```